src/tsdash.js

```diff
@@ -857,6 +889,74 @@ TsDashboard.prototype.drawTimeSeriesMulti = function (config) {
     }
 }
 
+TsDashboard.prototype.drawTable = function (config) {
+
+    var self = this;
+    // Default parameters.
+    var p = {
+        chart_div: "#someChart",
+        data: null,
+        header: null,
+        height: 400,
+        margin_bottom: 60,
+        column_widths: null
+    };
+    
+    // If we have user-defined parameters, override the defaults.
+    if (config !== "undefined") {
+        for (var prop in config) {
+            p[prop] = config[prop];
+        }
+    }
+
+    // remove the previous drawing
+    $(p.chart_div).empty();
+
+    // set style
+    $(p.chart_div).css('overflow', 'auto');
+    $(p.chart_div).css('height', p.height);
+    $(p.chart_div).css('margin-bottom', p.margin_bottom);
+    var column_widths = p.column_widths;
+    
+    var data = p.data[0];
+
+    // generate table html node
+    var table = $("<table class=\"table\"></div>");
+    
+    // fill table header
+    var thead = $("<thead></thead>");
+    var theadtr = $("<tr></tr>");
+    header = p.header;
+    if (!header) {
+        header = [];
+        for (let att in data[0]) {
+            header.push(att);
+        }
+    }
+    for (let h of header) {
+        theadtr.append("<td>"+h+"</td>");
+    }
+    table.append(thead.append(theadtr)); 
+
+    // fill table body 
+    var tbody = $("<tbody></tbody>");
+    for (let n of data) {
+        // create row
+        var row = $("<tr></tr>");
+        // add columns
+        for (att in n) {
```

Comment:
Here you have no control over order of data that is presented. I suggest we enhance the header-definition section. In the future.